vertical-pod-autoscaler/hack/run-integration-locally.sh

```
@@ -155,7 +153,7 @@ case ${SUITE} in
   recommender)
 
     export KUBECONFIG=$HOME/.kube/config
-    pushd ${SCRIPT_ROOT}/e2e
+    pushd ${SCRIPT_ROOT}/test/e2e
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)